docs/docs/Configuration.md

```diff
@@ -53,9 +104,12 @@ fsharp_strict_mode=false
 Please note that you should only add settings to the `.editorconfig` file when you want to deviate from the default settings.
 Copying the entire list above is unnecessary.
 
-### indent_size
+# Each setting explained
```

Comment:
I'm not really a fan of having multiple h1 headers on the same page.
We don't do this on the other pages, the h1 should be the title of the page and only occur once.
Please address.

Reply:
@nojaf, yes, that was my main point here. I didn't really like that either, but it originally rendered better with the ToC. I can remove `Configuration` from the ToC, but as you mentioned above, maybe there won't be a ToC at all. So either way, I'll resolve.